lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -102,6 +104,49 @@ sub register {
   );
 }
 
+sub _check_for_wildcards {
+    my ( $self, $files ) = @_;
+
+
+    #Keep track of files that we have seen already
+    my %seenFiles;
+    #We will replace the files array with this one
+    my @new_files_array;
+    #Loop through files
+    foreach my $file ( @{$files} ) {
+
+        #Check for *. files EX: *.js, *.css
```

Comment:
I wonder if this whole thing can be replaced with:

    @files = grep { !$seen{$_}++ } glob $file;